great_expectations/rule_based_profiler/parameter_builder/simple_dateformat_string_parameter_builder.py

```diff
@@ -4,22 +4,24 @@
 
 import great_expectations.exceptions as ge_exceptions
 from great_expectations import DataContext
-from great_expectations.execution_engine.execution_engine import MetricDomainTypes
 from great_expectations.rule_based_profiler.domain_builder.domain import Domain
 from great_expectations.rule_based_profiler.parameter_builder.parameter_builder import (
     ParameterBuilder,
 )
 from great_expectations.rule_based_profiler.parameter_builder.parameter_container import (
-    DOMAIN_KWARGS_PARAMETER_NAME,
     ParameterContainer,
     build_parameter_container,
 )
+from great_expectations.rule_based_profiler.util import (
+    get_parameter_value_and_validate_return_type,
+)
 from great_expectations.validator.validation_graph import MetricConfiguration
 from great_expectations.validator.validator import Validator
 
 logger = logging.getLogger(__name__)
 
 
+# TODO: <Alex>ALEX -- This ParameterBuilder example is stale (and would need work prior to using and/or testing).</Alex>
```

Comment:
@cdkini We need to leave this as is.  This parameter builder is out of scope, but I wanted to keep it in the codebase, because it provides a very useful example for the future, and I did not want to lose it (or have to look hard for it).  Thanks!